### PR TITLE
libsdl2: Fix build with libudev and joystick subsystem disabled

### PR DESCRIPTION
**Pull request by landgraf**
Upstream commit: 71fb91f7e43c5f046a037bf5ca59214d93fe2d51

### CODE REVIEW

File: media-libs/libsdl2/files/libsdl2-2.24.0-fix-build-without-joystick.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+From 71fb91f7e43c5f046a037bf5ca59214d93fe2d51 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline media-libs/libsdl2/files/libsdl2-2.24.0-fix-build-without-joystick.patch:1]: Please pop a link to the commit at the top of this file.

Comment (landgraf) [reply media-libs/libsdl2/files/libsdl2-2.24.0-fix-build-without-joystick.patch:2]: Should be fixed now

**Comment by landgraf** [comment]
> Thanks, please add signoff to your commit too.

SOB added.

**Comment by thesamesam** [comment]
Thanks!

**Pull request closed without merging**